stack-20.yaml

```diff
@@ -0,0 +1,13 @@
+resolver: lts-20.4
+allow-newer: true
```

Comment:
Why this?

Reply:
I copied it from `stack-19.yaml`.

I bumped the resolver to `20.4`, upgraded to `proto3-wire-1.4.0` and the commit hash of `http2-client`.